R/mia.R

```diff
@@ -202,3 +196,37 @@ NULL
 #' \url{https://doi.org/10.3390/ijerph18084049}
 #' 
 "HintikkaXOData"
+
+#' SilvermanAGutData
+#' 
+#' The SilvermanAGutData dataset contains 16S rRNA gene based high-throughput 
+#' profiling of 4 in vitro artificial gut models. The sampling was done hourly 
+#' and daily to capture sub-daily dynamics of microbial community originating 
+#' from human feces. The data consists of 413 taxa from 639 samples. 
+#' The data set can be used to investigate longitudinal dynamics of microbial 
+#' community in a controlled environment. 
```

Comment:
This is very nice time series data. Hence it would better with the miaTime package. Could we put it there instead?

Reply:
Sure, I can make a PR and just copy-paste these